prusti-tests/tests/parse/pass/basic_prusti_reentry.rs

```diff
@@ -1,6 +1,5 @@
 use prusti_contracts::*;
 
-// ignore-test: prusti-expr! not yet implemented
 #[requires(x == 1 || prusti_expr!(y == 0 ==> z == 0))]
```

Comment:
This should also not have `prusti_expr!` at all?

Reply:
We haven't yet implemented disjunctions as a native prusti operator

Reply:
Sure, but isn't the plan for re-entrancy to work based on parentheses rather than this explicit macro?

Reply:
Yes. My bad. Fixed by 681ccfe0b2aa35aa9ed35a05b55490ba69ad34b9